src/components/task/TaskPanel.tsx

```diff
@@ -1,9 +1,14 @@
-import { Card, CardActions, CardContent, Collapse, ExpansionPanelDetails, ExpansionPanelSummary, Grid, Typography } from '@material-ui/core';
+import { Card, CardActions, CardContent, Collapse, ExpansionPanelDetails, ExpansionPanelSummary, Grid, Link, Typography } from '@material-ui/core';
 import MuiExpansionPanel from '@material-ui/core/ExpansionPanel';
+import CalendarIcon from '@material-ui/icons/DateRangeOutlined';
```

Comment:
I think the icon imports can be consolidated. Can you try doing `import * as icons from '@material-ui/icons';` and then when you need to access them, do `icons.ExpandLess`

Reply:
Fixed